components/org.apache.stratos.load.balancer/src/main/java/org/apache/stratos/load/balancer/endpoint/TenantAwareLoadBalanceEndpoint.java

```diff
@@ -801,10 +804,11 @@ public void setSessionAffinity(boolean sessionAffinity) {
         private org.apache.axis2.clustering.Member currentMember;
         private Endpoint currentEp;
         private EndpointReference to;
-        private Map<String, Boolean> faultyMembers;
+        private Map<String, Boolean> faultyMembersIds;
```

Comment:
Thanks @sajithswa for fixing this! Shall we rename map variable faultyMembersIds to faultyMemberIds?